src/modes/easter_egg_hunt.cpp

```diff
@@ -154,6 +162,16 @@ void EasterEggHunt::collectedEasterEgg(const AbstractKart *kart)
     m_eggs_found++;
 }   // collectedEasterEgg
 
+//-----------------------------------------------------------------------------
+/** Called when a ghost kart has collected an egg.
+ *  \param world_id The world id of the ghost kart that collected an egg.
+ */
+void EasterEggHunt::collectedEasterEggGhost(int world_id)
+{
+    m_eggs_collected[world_id]++;
+    printf("Egg collected, new total is %d\n", m_eggs_collected[world_id]);
```

Comment:
printf should not be left in the code. If it's valuable logs, use our Log:: framework. If not valuable, remove

Reply:
Sorry, forgot to remove those.